core/lib/resources/presence/presence_manager.js

```diff
@@ -39,13 +40,30 @@ PresenceManager.prototype.setup = function() {
   });
 
   // save so you removeListener on destroy
-  this.sentryListener = function(sentry) {
+  this.sentryListener = function (sentry) {
+    if (manager.destroying) return;
```

Comment:
It's an implicit flag, and that's fine once there is a comment that explains its use as a flag where it matters.